wrapanapi/systems/msazure.py

```diff
@@ -669,6 +677,23 @@ def list_free_nics(self, nic_template, resource_group=None):
         ips = self.network_client.network_interfaces.list(resource_group_name=resource_group)
         return [ip.name for ip in ips if not ip.virtual_machine and nic_template in ip.name]
 
+    def list_all_free_nics(self, resource_group=None):
+        """
+        List all available Network Interfaces in selected resource_group
+        """
+        resource_group = resource_group or self.resource_group
+        nics = self.network_client.network_interfaces.list(resource_group_name=resource_group)
+        return [nic.name for nic in nics if not nic.virtual_machine]
+
+    def list_all_free_discs(self, resource_group=None):
```

Comment:
In keeping with the above function signatures, I think this should be `list_free_discs`